Marlin/Marlin_main.cpp

```diff
@@ -4567,7 +4567,7 @@ inline void gcode_M221() {
       SERIAL_PROTOCOLLN(extruder_multiplier[target_extruder]);
     }
     else {
-      SERIAL_PROTOCOL(int(code_value()));
+      SERIAL_PROTOCOL(int(active_extruder));
```

Comment:
As I said up in the pull request, noticed that I done goofed when I added these lines to display the extrusion multiplier a few months ago, wanted to fix it up here right quick.